modules/appeals_api/app/controllers/appeals_api/schemas/shared_schemas_controller.rb

```diff
@@ -8,8 +8,10 @@ class AppealsApi::Schemas::SharedSchemasController < AppealsApi::ApplicationCont
 
   ACCEPTED_SCHEMA_TYPES = %w[
     address
+    icn
     nonBlankString
     phone
+    ssn
```

Comment:
I forgot to add icn here as a valid shared schema (to make it possible to fetch it from the shared schemas endpoint) in my last PR, and I've added an ssn.json shared schema in this PR